types/react-navigation-fluid-transitions/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+      "no-any-union": false,
```

Comment:
Why is it necessary to disable these lint rules?

Reply:
Because when i tried to run the `yarn test` pops ups errors and i tried a lot to fix them so to don't waste time i disabled  those rules